src/aui-button/js/aui-button-core.js

```diff
@@ -106,6 +107,16 @@ ButtonExt.ATTRS = {
     iconAlign: {
         validator: A.Lang.isString,
         value: 'left'
+    },
+
+    /**
+     * Sets visible/hidden state.
+     *
+     * @default true
+     * @type {Boolean}
+     */
+    visible: {
```

Comment:
All yui widgets already have a `visible` attribute, which is true by default, so I think you shouldn't need to redeclare this here. Can you check if everything works without this? Same for button switch.